cves/CVE-2013-0868.yml

```diff
@@ -1,368 +1,406 @@
-  answer:
-  note:
-lessons:
-  question: |
-    Are there any common lessons we have learned from class that apply to this
-    vulnerability? In other words, could this vulnerability serve as an example
-    of one of those lessons?
-
-    Leave "applies" blank or put false if you did not see that lesson (you do
-    not need to put a reason). Put "true" if you feel the lesson applies and put
-    a quick explanation of how it applies.
-
-    Don't feel the need to claim that ALL of these apply, but it's pretty likely
-    that one or two of them apply.
-
-    If you think of another lesson we covered in class that applies here, feel
-    free to give it a small name and add one in the same format as these.
-  defense_in_depth:
-    applies:
-    note:
-  least_privilege:
-    applies:
-    note:
-  frameworks_are_optional:
-    applies:
-    note:
-  native_wrappers:
-    applies:
-    note:
-  distrust_input:
-    applies:
-    note:
-  security_by_obscurity:
-    applies:
-    note:
-  serial_killer:
-    applies:
-    note:
-  environment_variables:
-    applies:
-    note:
-  secure_by_default:
-    applies:
-    note:
-  yagni:
-    applies:
-    note:
-  complex_inputs:
-    applies:
-    note:
-mistakes:
-  question: |
-    In your opinion, after all of this research, what mistakes were made that
-    led to this vulnerability? Design mistakes? Maintainability? Requirements?
-    Miscommunications? Lack of testing? Lack of understanding? Lack of
-    specifications? Working alone?
-
-    An answer like "it was just a coding mistake" is not thoughtful enough.
-    If it's such an easy mistake to make, how was it missed?
-
-    Also, look at the CWE entry for this vulnerability and examine the mitigations
-    they have written there. Are they doing those? Does the fix look proper?
-
-    Use those questions to inspire your answer. Don't feel obligated to answer
-    every one. Write a thoughtful entry here that those ing the software
-    engineering industry would find interesting.
-  answer:
-CWE_instructions: |
-  Please go to http://cwe.mitre.org and find the most specific, appropriate CWE
-  entry that describes your vulnerability. We recommend going to
-  https://cwe.mitre.org/data/definitions/699.html for the Software Development
-  view of the vulnerabilities. We also recommend the tool
-  http://www.cwevis.org/viz to help see how the classifications work.
-
-  If you have anything to note about why you classified it this way, write
-  something in CWE_note. This field is optional.
-
-  Just the number here is fine. No need for name or CWE prefix. If more than one
-  apply here, then choose the best one and mention the others in CWE_note.
-CWE:
-CWE_note:
-nickname_instructions: |
-  A catchy name for this vulnerability that would draw attention it. If the
-  report mentions a nickname, e.g. "Heartbleed", use that. Or come up with one!
-
-  Must be under 30 characters. Optional. Be appropriate.
-nickname:
+CVE: CVE-2013-0868

```

Comment:
Not quite sure why github is saying that EVERY file was changed - might be a newline thing not sure. If the build passes we're fine.

Reply:
I noticed that this was odd, I'm assuming across some editor I used the line endings/feeds got changed. If I have time this weekend I'll see if I can fix this, but thankfully the build seems to be passing either way.